login-workflow/example/src/screens/CustomPage.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { useState } from 'react';
+import { useApp } from '../contexts/AppContextProvider';
+import {
+    DrawerLayout,
+    Drawer,
+    DrawerBody,
+    DrawerNavGroup,
+    DrawerNavItem,
+    DrawerHeader,
+    Spacer,
+    EmptyState,
+} from '@brightlayer-ui/react-components';
+import Box from '@mui/material/Box';
+import Event from '@mui/icons-material/Event';
+import Dashboard from '@mui/icons-material/Dashboard';
+import Notifications from '@mui/icons-material/Notifications';
+import Menu from '@mui/icons-material/Menu';
+import AppBar from '@mui/material/AppBar';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+import { useTheme } from '@mui/material/styles';
+import { useTranslation } from 'react-i18next';
+import * as Colors from '@brightlayer-ui/colors';
+import FormControl from '@mui/material/FormControl';
+import { MenuItem, Select, SelectChangeEvent } from '@mui/material';
+
+export const CustomPage: React.FC<React.PropsWithChildren> = () => {
+    const app = useApp();
+    const { t, i18n } = useTranslation();
+    const [open, setOpen] = useState(false);
+    const theme = useTheme();
+
+    const containerStyles = {
+        width: '100%',
+        height: `calc(100vh - ${theme.spacing(8)})`,
+        display: 'flex',
+        padding: 0,
+        overflow: 'auto',
+        position: 'relative',
+        flexDirection: 'column',
+        alignItems: 'center',
+        justifyContent: 'center',
+        [theme.breakpoints.down('sm')]: {
+            height: `calc(100vh - ${theme.spacing(7)})`,
+        },
+    };
+
+    const emptyStateContainerStyles = {
+        margin: 'auto',
+        display: 'flex',
+        zIndex: 4,
+        position: 'absolute',
+        alignItems: 'center',
+        justifyContent: 'center',
+    };
+
+    const changeAppLanguage = (event: SelectChangeEvent): void => {
+        const appLanguage = event.target.value;
+        app.setLanguage(appLanguage);
+        void i18n.changeLanguage(appLanguage);
+        window.localStorage.setItem('language', appLanguage);
+    };
+
+    return (
+        <DrawerLayout
+            drawer={
+                <Drawer open={true} width={332} variant={'persistent'}>
+                    <DrawerHeader
+                        title={`${t('DRAWER_MENU.TITLE')}`}
+                        icon={<Menu />}
+                        onClick={(): void => setOpen(!open)}
+                    />
+                    <DrawerBody>
+                        <DrawerNavGroup>
```

Comment:
It would be nice to have a way back to homepage...right now if you come to custompage, you can't go back to homepage and you can't log out, so you're stuck.